### PR TITLE
Update cli.py

### PR DESCRIPTION
**Pull request by giuliocalzolari**
fix typo

### CODE REVIEW

**Comment by giuliocalzolari** [comment]
@boroivanov

File: ssmrun/cli.py [hunk 1]
```diff
@@ -139,12 +139,12 @@ def docs(ctx, long_list, owner, platform, doc_version, doc_type, schema, profile
 
 
 @click.command()
-@click.argument('ssm-docutment')
+@click.argument('ssm-document')
 @click.option('-V', '--document-version', default=None, help='Document Version')
 @click.option('-p', '--profile', default=None, help='AWS profile')
 @click.option('-r', '--region', default=None, help='AWS region')
 def get(ssm_docutment, document_version, profile, region):
```
Comment (boroivanov) [inline ssmrun/cli.py]: @giuliocalzolari good catch. Please also update `ssm_docutment ` arg to `ssm_ document`

**Comment by giuliocalzolari** [comment]
done

File: ssmrun/cli.py (hunk 1 above)
Comment (giuliocalzolari) [reply ssmrun/cli.py]: done

**Pull request merged**